src/main/kotlin/com/zepben/evolve/services/network/NetworkServiceComparator.kt

```diff
@@ -245,6 +246,45 @@ class NetworkServiceComparator @JvmOverloads constructor(var options: NetworkSer
             compareIdReferenceCollections(OperationalRestriction::equipment)
         }
 
+    /************ IEC61968 infIEC61968 ************/
```

Comment:
nit: Can you please relocate this section to above `metering` to keep it alphabetical.